bin/snare

```diff
@@ -195,14 +195,14 @@ if __name__ == '__main__':
     if not check_meta_file(meta_info):
         print_color("Error found in meta.json. Please clone the pages again.", "ERROR")
         exit()
-
-    if not os.path.exists(os.path.join(base_page_path,
-                                       args.page_dir,
+    try:
+        if not os.path.exists(os.path.join(base_page_path, args.page_dir,
                                        os.path.join(meta_info[args.index_page]['hash']))):
-        print_color('can\'t create meta tag', 'WARNING')
-    else:
-        snare_helpers.add_meta_tag(
-            args.page_dir, meta_info[args.index_page]['hash'], config)
+            print_color('can\'t create meta tag', 'WARNING')
+        else:
+            snare_helpers.add_meta_tag(args.page_dir, meta_info[args.index_page]['hash'], config)
+    except KeyError as e:
+        print_color(e,"has not been created.")
```

Comment:
this error message doesn't give me any clue how I should proceed, could we make it more informative?

Reply:
When I check all the "/opt/snare/pages" files created , I found one file created with "www.collegesolution.co.in" which raises an KeyError, 
I have attached screenshot for that 
![Screenshot from 2020-03-17 17-05-54](https://user-images.githubusercontent.com/45192178/76853488-61d38000-6873-11ea-9a2a-6777e43b1b54.png)

Reply:
Thanks for sharing! I meant that it would be nice to have more informative error message :)